README.md

```diff
@@ -74,33 +74,39 @@ You'll need a locally running Docker and Docker Compose:
 
 ### Quickstart
 
+Modify or create `.env.local`. This config uses the database container defaults.
+```
+DATABASE_URL=mysql://root:@database:3306/wsexport?serverVersion=5.7
+```
+
+Make sure you cd into `./docker`
+
+```bash
+cd ./docker 
+```
+
 Run the following command to add your user ID and group ID to your `.env` file:
 
 ```bash
-echo "WS_DOCKER_PORT=8888
-WS_DOCKER_UID=$(id -u)
-WS_DOCKER_GID=$(id -g)" >> .env.local
+echo "WS_DOCKER_UID=$(id -u)
+WS_DOCKER_GID=$(id -g)" >> ./.env
 ```
 
-#### Start environment and install
+Start the environment and install
 
 ```bash
 # -d is detached mode - runs containers in the background:
-docker-compose --env-file=.env.local up -d
+docker-compose build && docker-compose up -d
```

Comment:
Trying a second time, here I get:
```
WARNING: The WS_DOCKER_PORT variable is not set. Defaulting to a blank string.
ERROR: The Compose file './../docker-compose.yml' is invalid because:
services.wsexport.ports contains an invalid type, it should be a number, or an object
```

I am using docker version 19.03.13 and docker-compose version 1.25.0, if that matters.